container/build/system-packages

```diff
@@ -0,0 +1,35 @@
+build-essential
+bundler
+cmake
+curl
+gettext
+ghostscript
+git
+language-pack-en
+libavcodec-dev
+libavformat-dev
+libevent-dev
+libffi-dev
+libgtk2.0-dev
+libjpeg-dev
+libpq-dev
+libssl-dev
+libxml2-dev
+libxslt-dev
+libyaml-dev
+npm
+opencv-data
+postgis
```

Comment:
I'm pretty sure there are some packages in here we don't need installed in the frontend container. For example, `postgis` immediately jumps out at me. Also some of these `python3-` packages might be drop-able. Lets not get too bogged down in this right now. We can prune it another day (and it is probably better for someone who knows the application well to do it), but just flagging while it jumps out at me.

Reply:
Agreed - Sym and I had identified that there would probably be some accumulated cruft in the current deployment's package set, but given our phase-2 aim is to deploy a production container setup as similar as possible to the current EC2 instance and not spend time troubleshooting non-essential deltas, it felt appropriate to cargo cult the package set initially.